### DISCUSSION TITLE
How to ignore selected pages from having parent layouts?

### DISCUSSION THREAD

**Discussion by bgrgv** [vercel/next.js#55939] [App Router] [answered]
I'm using Next v 13.5.2 and have a doubt regarding ignoring selected pages from having parent layouts.

Consider the following URLs:

<img width="178" alt="image" src="https://github.com/vercel/next.js/assets/46672514/9a04b665-86be-4241-938e-bae78f5f30d5">

I want to avoid `layout.js` from `app/products/gold/` from being applied to `app/products/gold/help/layout.js`.

Restructuring is a possible way, but is there a different way to ignore a specific parent layout from a specific page in one of the subdirectories?

**Comment by icyJoseph** [comment]
I think the way to go is to use https://nextjs.org/docs/app/building-your-application/routing/route-groups

**Reply by neeraj15022001** [reply]
Is there a way without router groups like overriding parent layouts?

**Reply by icyJoseph** [reply, marked as answer]
As far as I know, nope. If its not in the docs, then there probably isn't any.
